ClosedXML/Excel/Cells/XLCell.cs

```diff
@@ -356,13 +356,21 @@ public string GetFormattedString()
                 return cValue;
         }
 
+        /// <summary>
+        /// Flag showing that the cell is in formula evaluation state.
+        /// </summary>
+        internal bool IsEvaluating { get; private set; }
```

Comment:
Would you mind submitting the `XLCell` changes as a separate PR? I'll merge that first.